glnx-errors.h

```diff
@@ -109,7 +109,8 @@ typedef struct {
 static inline void
 glnx_cleanup_auto_prefix_error (GLnxAutoErrorPrefix *prefix)
 {
-  g_prefix_error (prefix->error, "%s: ", prefix->prefix);
+  if (prefix->error && *(prefix->error))
```

Comment:
It's definitely true that other cleanups do things like this; but it's also kind of silly, `g_autoptr_cleanup_generic_gfree()` also does a `NULL` check, then so does `g_free()`, then so does `free()`...